pkg/db/fetch/fetch.go

```diff
@@ -135,9 +149,44 @@ func Fetch(opts ...Option) error {
 	}
 	defer f.Close()
 
-	if _, err := d.WriteTo(io.MultiWriter(f, progressbar.DefaultBytes(-1, "downloading"))); err != nil {
+	pb := func() *progressbar.ProgressBar {
+		if options.noProgress {
+			return progressbar.DefaultBytesSilent(-1)
+		}
+		return progressbar.DefaultBytes(-1, "downloading")
+	}()
+	defer pb.Finish()
```

Comment:
use https://pkg.go.dev/github.com/schollz/progressbar/v3#ProgressBar.Close instead of `Finish()`
```suggestion
	defer pb.Close()
```